pkg/statistics/handle/globalstats/globalstats_test.go

```diff
@@ -76,6 +76,24 @@ func TestGlobalStatsPanicInIOWorker(t *testing.T) {
 	simpleTest(t)
 }
 
+func TestGlobalStatsWithCMSketchErr(t *testing.T) {
```

Comment:
Please add 2 more test cases:
1. both IOWorker and CPUWorker error
2. both IOWorker and CPUWorker panic

Reply:
Done